ts/output/common/Wrappers/mrow.ts

```diff
@@ -291,6 +302,26 @@ export function CommonMrowMixin<
       bbox.clean();
     }
 
+    /**
+     * Adjust bbox vertical alignment. (E.g., for \vbox, \vcenter.)
+     */
+    protected vboxAdjust(bbox: BBox) {
+      const n = this.breakCount;
+      const valign = this.parent.node.attributes.get('data-vertical-align');
+      if (valign === 'bottom' && n) {
+        this.dh = (n ? bbox.d - this.lineBBox[n - 1].d : 0);
+      } else if (valign === 'center' || (valign === 'middle' && n)) {
```

Comment:
Why don't we need a breakcount on `center`?
Also swap `n`.

Reply:
As I mention in the initial PR discussion, `center` is centered regardless of linebreaks.  This is what `\vcenter` does, whereas `middle` only centers if there are line breaks.  Without this distinction, you would need to keep the `\vcenter` code in the `TeXAtom` node, which was always a bit out of place there but was there because I was trying to keep all the non-MathML functionality in one place.  We now have various extensions to MathML attributes (e.g., `data-padding` for `menclose`, and this `data-vertical-align`, and a number of table border extensions, etc.), so that isolation of non-MathML material no longer holds, and it seemed better to go back to `TeXAtom` just handling the change of `texClass`.

Reply:
Right. Sorry about not making the connection. I had read the explanation when I started the review, but by the time I looked at this file (after lunch...) the information had left my brian.